### PR TITLE
chore: update readme

### PR DESCRIPTION
**Pull request by andersfylling**

### CODE REVIEW

**Comment by ebrattli** [comment]
why?

**Comment by andersfylling** [comment]
No need for both of them. The `--since master` will always execute tests for all affected packages. In essence only executing the fewest tests possible to verify changes. While running all tests may only add redundant information on top of using `--since master`.

File: README.md [hunk 1]
```diff
@@ -43,16 +43,7 @@ This repo contains some integration tests that require a CDF api key for `cognit
...
-Run all tests:
-
-```bash
-yarn
-yarn build
-yarn test
-```
-
-Run tests for updated package only:
-> for changes to the core package, it might be required to run tests for all packages to verify integrity.
```
Comment (ebrattli) [inline README.md:55]: This might be useful to know.

Comment (andersfylling) [reply README.md:55]: if the core has changed, then any other packages that depends on core are marked "dirty". So all tests will be run.

**Review by ebrattli — APPROVED** [review]
Ok. LGTM!

**Pull request merged**